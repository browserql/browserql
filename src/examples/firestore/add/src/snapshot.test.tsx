import result from './index'

it('renders correctly', async () => {
  const r = await result()
  delete r.data.firestoreAdd_Todo.id
  expect(r).toMatchSnapshot()
})
