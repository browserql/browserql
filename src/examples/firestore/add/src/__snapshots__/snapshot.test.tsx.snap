// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "data": Object {
    "firestoreAdd_Todo": Object {
      "__typename": "Todo",
      "done": false,
      "title": "Buy milk",
    },
  },
}
`;
